ci/ci_acceptance.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+echo "Building Logstash artifacts"
+rake artifact:all
+
+echo "Acceptance: Installing dependencies"
+cd qa
+bundle install
+
+echo "Acceptance: Running the tests"
+bundle exec rake test:setup
+bundle exec rake test:ssh_config
+bundle exec rake test:acceptance:all
```

Comment:
we need it to get test that run for a given platform, it will give us
faster and detailed feedback in comparison to run all at once.

On Thu, Apr 28, 2016 at 4:30 PM Pier-Hugues Pellerin <
notifications@github.com> wrote:

> In ci/ci_acceptance.sh
> <https://github.com/elastic/logstash/pull/5218#discussion_r61436646>:
>
> > @@ -0,0 +1,15 @@
> > +#!/bin/sh
> > +set -e
> > +
> > +echo "Building Logstash artifacts"
> > +rake artifact:all
> > +
> > +echo "Acceptance: Installing dependencies"
> > +cd qa
> > +bundle install
> > +
> > +echo "Acceptance: Running the tests"
> > +bundle exec rake test:setup
> > +bundle exec rake test:ssh_config
> > +bundle exec rake test:acceptance:all
>
> Why would we need a platform type here?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/5218/files/44dab96d1bafb372ed6271e9bf91933b36bb4c46#r61436646>
>